app/views/categories/_categories.html.haml

```diff
@@ -0,0 +1,18 @@
+%ul.category-summary
+  - categories.each do |category|
+    - if category.has_children?
+      - count = category.children.inject(0) do |memo, child|
```

Comment:
This is the expression on the inside of deep loop.
So I want to reduce loop here.